java/ql/src/Security/CWE/CWE-200/TempDirUtils.qll

```diff
@@ -0,0 +1,83 @@
+import java
```

Comment:
Needs file-level qldoc. Should be on the form "Provides ...". For example "Provides classes and predicates for reasoning about temporary directory creations." or something along those lines (possibly my example qldoc is a bit too specific?).